internal/analytics/segment_reporter.go

```diff
@@ -106,6 +113,25 @@ func (r *SegmentReporter) ReportBotEnabled(platform config.CommPlatformIntegrati
 	})
 }
 
+func (r *SegmentReporter) ReportPluginsConfig(executors map[string]config.Executors, sources map[string]config.Sources) error {
+	pluginsConfig := make(map[string]interface{})
+	for _, values := range executors {
+		for name, pluginValue := range values.Plugins {
+			if pluginValue.Enabled {
```

Comment:
You can invert the condition and in that way make the code simpler (with less indentation).

```go
if pluginValue.Disabled {
    continue
}
```

Change it in both places.